README.md

```diff
@@ -169,3 +169,29 @@ proxy_service:
 ```
 
 Then start the dev server like `yarn start-teleport --target=https://proxy.127.0.0.1.nip.io:3080` and access it at https://proxy.127.0.0.1.nip.io:8080.
+
+### Adding Packages with `yarn`
```

Comment:
I'm can't tell what about the `build/README` needs updating. I think these instructions belong out here, because they apply to all of the sub workspaces, not just to `build/`.

Reply:
sorry i meant `packages/README` (the link that i linked correctly, but named incorrectly)